src/queries.js

```diff
@@ -1,11 +1,25 @@
 // TODO: Sanitize string input of buckets
 
+// Helper function returning a query that defines a variable events_active
+// that can be used to filter away non-active time.
+function events_active(afkbucket, audibleAsActive) {
+  return [
+    'events_active = [];',
```

Comment:
Why empty array and perdiod_union?
why not just `events_active = flood(query_bucket("${afkbucket}"));`?